src/mono/netcore/sample/wasm/Makefile

```diff
@@ -13,7 +13,7 @@ clean:
 	rm -rf bin
 
 run:
-	cd bin/$(CONFIG)/publish && ~/.jsvu/v8 --expose_wasm runtime.js -- --enable-gc --run WasmSample.dll
+	cd bin/$(CONFIG)/publish && ~/.jsvu/v8 --expose_wasm runtime.js -- --enable-gc --enable-zoneinfo --run WasmSample.dll 
```

Comment:
This would also break the sample right? Does the sample have the `supportFiles`?

Reply:
Oh nevermind, I see you're adding a package reference to it.